tools/releasing/packaging/docker/README.md

```diff
@@ -2,6 +2,8 @@
 
 The point of these dockerfiles is to build a Ubuntu-based distribution of preCICE.
 
+All images contain the user `precice`, allowing them to run executables using MPI.
```

Comment:
Could we add such a README file on Docker Hub, explaining what each image is for?